src/pydata_sphinx_theme/assets/styles/components/_sidebar_toggle.scss

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
```

Comment:
Comment for above (the diff is hidden)

I'm not sure why this commit registers as adding a `package-lock.json` file, I thought that's already on `main`, can you double check that?

Reply:
I created and merged the package-lock.json file on the 11th of June, this branch was created on the 10th. I think it was identified as a new file when I switched back to this one and was accidentally added to the branch. They are identical. The problem should solve itself on merge.